src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py

```diff
@@ -266,6 +266,11 @@ def load_arguments(self, _):
                    help='Provide a string value of a Storage Account in the provided Resource Group. Or Resource ID of a Storage Account in a different Resource Group')
         c.argument('consumption_plan_location', options_list=['--consumption-plan-location', '-c'],
                    help="Geographic location where Function App will be hosted. Use 'functionapp list-consumption-locations' to view available locations.")
+        c.argument('is_linux', action='store_true', required=False, help='host webapp on Linux worker')
+        c.argument('zip_url', help='Url string to zip location The content from this location is deployed to the function app')
+        c.argument('runtime', help='The linux runtime stack', arg_type=get_enum_type(['dotnet', 'javascript', 'python']))
```

Comment:
should be the function runtime stack. also it should be `node` not `javascript`

Reply:
ok will do that, given this is currently supported to Linux only - will add currently supported for Linux only?